packages/xo-server/sample.config.toml

```diff
@@ -152,6 +152,11 @@ port = 80
 #
 # Default: '/run/xo-server/mounts'
 #mountsDir = '/run/xo-server/mounts'
+#
+# timeout before reporting servers issues
+#
+# Default: '5 minutes'
+#noEventsTimeout = '5 minutes'
```

Comment:
Not necessary, as I told you before, this is an advanced setting and we don't expect user to change it, therefore it should not be documented in the sample config.